src/base64.cr

```diff
@@ -153,7 +153,7 @@ module Base64
     strict_encode_to_io_internal(data, io, CHARS_SAFE, pad: true)
   end
 
-  # Returns the Base64-decoded version of `data` as a *Bytes*.
+  # Returns the Base64-decoded version of `data` as a `Bytes`.
```

Comment:
I've changed it earlier but there are multiple instances of such notation so I could replace throughout the file if needed.